File: Java_Practice/Python/struct_access_modify.py
#Access & modify members using reference
class Student:
    def __init__(self, name, roll, marks):
        self.name = name
        self.roll = roll
        self.marks = marks

s = Student("Ravi", 101, 88.5)

print("Original Name:", s.name)

# Modify using reference
s.name = "Kiran"
s.marks = 95.0

print("Modified Name:", s.name, ", Marks:", s.marks)

File: Java_Practice/CPP/struct_access_modify.cpp
//Access & modify structure members using pointer
#include <iostream>
using namespace std;

struct Student {
    string name;
    int roll;
    float marks;
};

int main() {
    Student s = {"Ravi", 101, 88.5};
    Student *ptr = &s;

    cout << "Original Name: " << ptr->name << endl;

    ptr->name = "Kiran"; // Modify using pointer
    ptr->marks = 95.0;

    cout << "Modified Name: " << ptr->name << ", Marks: " << ptr->marks << endl;

    return 0;
}
